poetry.lock

```diff
@@ -37,9 +37,6 @@ optional = false
 python-versions = "*"
 version = "3.2.3"
 
-[package.extras]
```

Comment:
Please update your local version of poetry and relock - these lines were added in the lock file when we upgraded to Poetry >1.0.0

Reply:
Please fix this, not resolved